bindata/etcd/etcd-snapshot-backup.sh

```diff
@@ -12,13 +12,33 @@ if [[ $EUID -ne 0 ]]; then
   exit 1
 fi
 
-usage () {
+function usage {
     echo 'Path to backup dir required: ./etcd-snapshot-backup.sh <path-to-backup-dir>'
     exit 1
 }
 
-ASSET_DIR=/home/core/assets
+#backup latest static pod resources for kube-apiserver
+function backup_latest_kube_static_resources {
+  echo "Trying to backup latest static pod resources.."
+  LATEST_STATIC_POD_DIR=$(ls -vd "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]* | tail -1) || true
```

Comment:
`CONFIG_FILE_DIR` is unbound here, isn't it?

Reply:
you need to source the value I guess.

Reply:
I see you sourced below function is called

Reply:
The source is below at L58. It worked fine, as the shell evaluates variables at the time of invocation.
https://github.com/openshift/cluster-etcd-operator/blob/00c5f417617f9a2c008d8a3cad957c71406dae32/bindata/etcd/etcd-snapshot-backup.sh#L58

Reply:
If you want, I can move it above.